apps/miniapp/domains/condo/constants/user.js

```diff
@@ -0,0 +1,11 @@
+const RESIDENT_USER_TYPE = 'resident'
+const STAFF_USER_TYPE = 'staff'
+const SERVICE_USER_TYPE = 'service'
```

Comment:
It's the same as
apps/miniapp/domains/condo/constants/common.js

Reply:
Yes, but it's miniapp costs %( 
I don't think that it's a good idea to use something from another app